src/app/ui/user-profile/user-profile.component.html

```diff
@@ -12,7 +12,13 @@
             <img mat-card-avatar class="avatar"
               [src]="user.photoURL || 'https://api.adorable.io/avatars/109/fire.png'"/>
             <mat-card-title>{{ user.displayName }}</mat-card-title>
-            <mat-card-subtitle>{{ user.membership?.teamName || 'No Team Yet' }}</mat-card-subtitle>
+            <!-- Players get a "No Team Yet" subtitle, while approvers just get blank space -->
+            <mat-card-subtitle>
+              {{ user.playerMembership?.teamName }} 
```

Comment:
since we are already getting just the playerMembership, this could be simply:
```suggestion
              <mat-card-subtitle>{{ user.playerMembership?.teamName || 'No Team Yet' }}</mat-card-subtitle>
```

Reply:
Oh, I only did that convoluted code snippet because I thought it had to be a blank space instead of "No Team Yet" if an approver was viewing his own profile due to the pics. But if it weren't the case, you're right, that's much simpler. Thanks!